src/components/menu/Menu.tsx

```diff
@@ -25,8 +32,14 @@ interface IMenuProps {
     visible,
     locationX,
     locationY,
+    range: state.textSelector.range,
   };
-})
+},
+  dispatch => ({
```

Comment:
if you just want to wrap actions into dispatch, it's possible to just pass an object here, like:
```
{
  showEditor: showEditorNewAnnotation,
  hideMenu,
}
```

`connect` automatically wraps them in `dispatch` (not sure about the 1st one, but I guess it passes arguments to the wrapped function)

Reply:
Good to know, I'll check

Reply:
I tried with `hideMenu` and it stops working. Are you sure about this shortcut?

Reply:
like so?
```
@connect( state => ({
  ...
}), {
  hideMenu,
  otherFn,
})
```

Reply:
Like so:
```
showEditor: (x, y, range) => dispatch(showEditorNewAnnotation(x, y, range)),
hideMenu,
```
and
```
showEditor: (x, y, range) => dispatch(showEditorNewAnnotation(x, y, range)),
hideMenu: hideMenu(),
```